src/repr/src/row/encoding.rs

```diff
@@ -527,6 +625,12 @@ impl<'a, T: DatumToPersist> DatumDecoderT<'a> for DataRef<'a, T> {
     }
 }
 
+impl DatumDecoderT<'_> for NoStats {
+    fn decode(&self, _idx: usize, _row: &mut RowPacker) {
+        unreachable!("the schema'd encoding is not yet used for: {:?}", self.0)
```

Comment:
Am I understanding right that this will throw if we ever attempt to decode with a schema that includes one of these types, and this is safe because we never run the decode path currently?